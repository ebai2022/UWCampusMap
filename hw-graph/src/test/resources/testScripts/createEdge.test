# Test edge creation

# Test single case
CreateGraph A
AddNode A e1
AddNode A e2
AddEdge A e1 e2 test