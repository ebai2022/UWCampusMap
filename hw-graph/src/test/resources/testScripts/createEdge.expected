# Test edge creation

# Test single case
created graph A
added node e1 to A
added node e2 to A
added edge test from e1 to e2 in A